nfd-worker-daemonset.yaml.template

```diff
@@ -61,6 +61,10 @@ spec:
             - name: nfd-worker-conf
               mountPath: "/etc/kubernetes/node-feature-discovery"
               readOnly: true
+## Example for more custom configs in an additional configmap (1/3)
+#            - name: more-custom-configs
+#              mountPath: "/etc/kubernetes/node-feature-discovery/custom.d/more-custom-configs"
```

Comment:
```suggestion
#              mountPath: "/etc/kubernetes/node-feature-discovery/custom.d"
```
just like this, kubernetes will mount the configMap data at `/etc/kubernetes/node-feature-discovery/custom.d/custom.conf`

Reply:
oh yes, thanks, in the e2e test I do it correct already :grin: